Library/ViewModels/CommentRepliesViewModel.swift

```diff
@@ -162,9 +162,21 @@ public final class CommentRepliesViewModel: CommentRepliesViewModelType,
         CommentsViewModel.postCommentProducer
       )
 
-    let repliesAndTotalCount = replies.withLatestFrom(totalCountProperty.signal)
+    let errorLoadingMoreReplies =
+      error
+        .withLatestFrom(totalCountProperty.signal)
+        .combineLatest(with: project)
+        .map { errorAndTotalCount, project -> (([Comment], Int), Project) in
```

Comment:
If we go with the other suggestion to call that function directly on the datasource then we shouldn't need this I think?

Reply:
Yup I re-factored to have a separate output handle the error. It's directly calling the showPaginationErrorState() method from the data source.